tsconfig.json

```diff
@@ -27,5 +27,5 @@
     "typeRoots": ["./node-modules/@types", "src/common/typedefs"]
   },
   "include": ["src"],
-  "exclude": ["vite.config.ts", "**/__tests__/**", "**/__mocks__/**"]
+  "exclude": ["vite.config.ts", "**/__mocks__/**"]
```

Comment:
You could just get rid of the whole `exclude` key here, along with deleting `tsconfig.eslint.json` (and pointing ESLint to this config file instead)